osu.Game/OsuGame.cs

```diff
@@ -102,12 +103,14 @@ protected override void Load(BaseGame game)
             });
 
             (Chat = new ChatConsole(API)).Preload(game, Add);
+            (MusicController = new MusicController()).Preload(game, Add);
 
             (Toolbar = new Toolbar
             {
                 OnHome = delegate { MainMenu?.MakeCurrent(); },
                 OnSettings = Options.ToggleVisibility,
                 OnPlayModeChange = delegate (PlayMode m) { PlayMode.Value = m; },
+                OnMusicController = MusicController.ToggleVisibility

```

Comment:
this binding never gets passed on to the toolbarbutton inside due to it being set in object initializer here (but passed on in ctor now)